omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -759,7 +759,7 @@ def apply_rdefs(self, image, channels):
                 c_idxs.append(i+1)
                 windows.append([c['window']['start'], c['window']['end']])
                 colors.append(c['color'])
-                reverses.append(c['reverseIntensity'])
+                reverses.append(c.get('reverseIntensity', False))
```

Comment:
Generally  :+1: for generalizing the use of `get(value, default)` for an optional attribute.

Beyond the scope of this fix, as we might be introducing more non-breaking functionalities like `reverseIntensity`, we might have to think on how best to codify this knowledge using some formal schema/specification so that others can consume the OMERO.figure format/API and know what to expect.